diff --git a/node_modules/react-native-pell-rich-editor/src/editor.js b/node_modules/react-native-pell-rich-editor/src/editor.js
index 4252bd9..6dab669 100644
--- a/node_modules/react-native-pell-rich-editor/src/editor.js
+++ b/node_modules/react-native-pell-rich-editor/src/editor.js
@@ -316,7 +316,9 @@ function createHTML(options = {}) {
                 result: function(url, style) {
                     if (url){
                         exec('insertHTML', "<img style='"+ (style || '')+"' src='"+ url +"'/>");
-                        Actions.UPDATE_HEIGHT();
+                        setTimeout(() => {
+                            Actions.UPDATE_HEIGHT();
+                        }, 50);
                     }
                 }
             },
@@ -324,7 +326,9 @@ function createHTML(options = {}) {
                 result: function (html){
                     if (html){
                         exec('insertHTML', html);
-                        Actions.UPDATE_HEIGHT();
+                        setTimeout(() => {
+                            Actions.UPDATE_HEIGHT();
+                        }, 50);
                     }
                 }
             },
@@ -335,7 +339,9 @@ function createHTML(options = {}) {
                         var thumbnail = url.replace(/.(mp4|m3u8)/g, '') + '-thumbnail';
                         var html = "<br><div style='"+ (style || '')+"'><video src='"+ url +"' poster='"+ thumbnail + "' controls><source src='"+ url +"' type='video/mp4'>No video tag support</video></div><br>";
                         exec('insertHTML', html);
-                        Actions.UPDATE_HEIGHT();
+                        setTimeout(() => {
+                            Actions.UPDATE_HEIGHT();
+                        }, 50);
                     }
                 }
             },
@@ -362,7 +368,7 @@ function createHTML(options = {}) {
             },
             content: {
                 setDisable: function(dis){ this.blur(); editor.content.contentEditable = !dis},
-                setHtml: function(html) { editor.content.innerHTML = html; Actions.UPDATE_HEIGHT(); },
+                setHtml: function(html) { editor.content.innerHTML = html; setTimeout(() => Actions.UPDATE_HEIGHT(), 50);  },
                 getHtml: function() { return editor.content.innerHTML; },
                 blur: function() { editor.content.blur(); },
                 focus: function() { focusCurrent(); },
